cpp/installer.cpp

```diff
@@ -7,32 +7,71 @@
  * This code is licensed under the MIT license
  */
 
-#include "react-native-quick-sqlite.h"
-#include "sequel.h"
+#include "installer.h"
+#include "sqliteBridge.h"
 #include "logs.h"
-#include "SequelResult.h"
+#include "JSIHelper.h"
+#include "ThreadPool.h"
 #include <vector>
 #include <iostream>
-#include <thread>
-#include <iostream>
 #include <fstream>
+#include <string>
 
 using namespace std;
 using namespace facebook;
 
-const vector<string> mapParams(jsi::Runtime &rt, jsi::Array &params)
+string docPathStr;
+std::shared_ptr<react::CallInvoker> invoker;
+
+/**
+ * Local function to handle SQL File Import in order to reuse with Sync and Async operations
+ */
+SequelBatchOperationResult importSQLFile(string dbName, string fileLocation)
```

Comment:
Maybe move this to it's own file?

Reply:
Do you mean include it on sequel.cpp?

Reply:
no, a separate file, the code is sqliteBridge deals with SQLite itself, I would like to keep the `installer` file simple with only the bindings. This metho deals mostly with IO operations, so I think it makes sense to keep it separate

Reply:
> no, a separate file, the code is sqliteBridge deals with SQLite itself, I would like to keep the `installer` file simple with only the bindings. This metho deals mostly with IO operations, so I think it makes sense to keep it separate

Its a nice idea. I will do that.